style.css

```diff
@@ -5,189 +5,205 @@
 }
 /*CSS RESET*/
 
+@font-face {
+  font-family: "AVGARDD";
+  src: url(/fonts/AVGARDD.TTF);
+}
+@font-face {
+  font-family: "AVGARDN";
+  src: url(/fonts/AVGARDN.TTF);
+}
+@font-face {
+  font-family: "impact_imp";
+  src: url(/fonts/impact.ttf);
+}
+body{
+  display:flex;
+  flex-direction: column;
+  background-color: #efefef
+}
 .header {
   height: 285px;
-  width: 100%;
   background-image: url(images/header-bg.jpg);
 }
-.header-content {
+.header__content {
   display: flex;
   justify-content: center;
   align-items: end;
 }
-.header-content span {
-  display: inline-block;
-}
 .logo {
+  position:relative;
+  left:-6px;
   background-image: url(images/logo.gif);
-  width: 372px;
+  background-repeat: no-repeat;
+  width:50%;
   height: 91px;
-  margin-left: 30px;
 }
-.pagetitle {
+.page__title {
   background-image: url(images/header-img1.jpg);
-  width: 311px;
+  background-repeat: no-repeat;
+  background-position: 90%;
+  width:50%;
   height: 215px;
 }
-.menubar {
+.menu__bar{
   display: flex;
-  height: 32px;
-  justify-content: center;
-  align-items: end;
-  position: absolute;
-  top: 215px;
-}
-.menubar ul {
-  list-style-type: none;
 }
-.menubar li {
-  float: left;
+.menu__items{
+  width:50%;
 }
 
-.btndivider {
-  vertical-align: middle;
-  background-image: url(images/menu-devider.gif);
-  background-repeat: no-repeat;
-  display: inline-block;
-  width: 2px;
-  height: 32px;
-}
-.hidden {
-  background-image: none;
-}
-.menubar a {
+.menu__items ul{
+  display: flex;
+  justify-content: space-between;
+  list-style: none;
+  width:360px;
+  height:32px;
+  white-space: nowrap;
+}
+.menu__items li{
+  height:32px;
+}
+.menu__items a{
+  display:inline-block;
+  line-height:32px;
+  text-align: center;
+  width:100%;
   font-family: "Trebuchet MS", sans-serif;
   font-size: 10pt;
   font-weight: bold;
   text-decoration: none;
   color: white;
-  margin: 0 15px 0 15px;
 }
-.menubar li:first-child a:first-of-type {
-  margin: 0 15px 0 8px;
+
+.menu__divider {
+  background-image: url(images/menu-devider.gif);
+  background-repeat: no-repeat;
+  display: inline-block;
+  width: 2px;
+  height: 32px;
 }
-.menubar li:hover {
+
+.menu__bar a:hover {
   background-image: url(images/menu-hover.gif);
   background-repeat: no-repeat;
   background-position: top;
 }
-.mainbody {
-  width: 100%;
-  height: 800px;
-  background-color: rgb(239, 239, 239);
-  position: relative;
+.main__body {
+  display: flex;
+  justify-content: center;
+  height:700px;
 }
-.bodycontent {
+.body__content {
   display: flex;
-  position: absolute;
-  width: 744px;
-  height: 800px;
-  left: 50%;
-  top: -35px;
-  transform: translateX(-50%);
+  width: 720px;
 }
 .column {
-  width: 317px;
-  height: 800px;
-  padding: 20px 0 0 45px;
+  width: 50%;
+  height:700px;
 }
 .column.right {
+  position:relative;
+  top:-33px;
```

Comment:
Nu inteleg de ce a trebuit sa folosesti `-33px` aici.

Poti sa scapi the `justify-content: space-evenly` sub `.column__content__container` si sa adaugi `.column__content__container > * { margin-top:  40px}` care va aplica 40px la marginea de sus pentru toti copiii lui `.column__content__container`.